### PR TITLE
Sort fields in browse mode, always show name and description in edit mode

### PR DESCRIPTION
**Pull request by makeitrein**

### CODE REVIEW

File: apps/web/modules/components/entity/edit-events.ts [hunk 1]
```diff
@@ -350,18 +350,6 @@ const listener =
...
-            {
-              ...triple,
-              entityName: value,
-              placeholder: false,
-              value: { ...triple.value, type: 'string', value },
-            },
-            triple
-          );
-        }
-
```
Comment (baiirun) [inline apps/web/modules/components/entity/edit-events.ts:364]: Are we sure this doesn't break anything for column editing?

File: apps/web/modules/components/entity/editable-fields.tsx [hunk 2]
```diff
@@ -29,6 +29,10 @@ export function StringField({ variant = 'body', ...props }: StringFieldProps) {
   const [localValue, setLocalValue] = React.useState(props.value || '');
   const ref = useRef<HTMLTextAreaElement>(null);
 
+  useEffect(() => {
+    setLocalValue(props.value || '');
+  }, [props.value]);
+
```
Comment (baiirun) [inline apps/web/modules/components/entity/editable-fields.tsx:35]: Are we sure we need this?

Comment (makeitrein) [reply apps/web/modules/components/entity/editable-fields.tsx:35]: Yup, otherwise the value is cached on initial mount.

Comment (makeitrein) [reply apps/web/modules/components/entity/editable-fields.tsx:35]: But now that I typed that out, let me double check...

File: apps/web/modules/components/entity/edit-events.ts (hunk 1 above)
Comment (baiirun) [reply apps/web/modules/components/entity/edit-events.ts:364]: Ah I think at some point I was using `UPDATE_VALUE` for column name editing but I guess I changed it to `EDIT_ENTITY_NAME`

Comment (makeitrein) [reply apps/web/modules/components/entity/edit-events.ts:364]: Bingo! I think we can merge this come Monday

File: apps/web/modules/components/entity/editable-fields.tsx (hunk 2 above)
Comment (makeitrein) [reply apps/web/modules/components/entity/editable-fields.tsx:35]: Yup, we need a useEffect to reset the input when the value changes from outside the component. Just asked Yaniv and Nate if this is desired behavior, feels a little strange to me atm.

File: apps/web/modules/components/entity/readable-entity-page.tsx [hunk 3]
```diff
@@ -92,7 +95,7 @@ export function ReadableEntityPage({ triples, id, name, space, linkedEntities }:
 
       <Content>
         <Attributes>
-          <EntityAttributes entityId={id} triples={triples} space={space} />
+          <EntityAttributes entityId={id} triples={sortedTriples} space={space} />
```
Comment (baiirun) [inline apps/web/modules/components/entity/readable-entity-page.tsx:98]: What is this change doing?

**Review by baiirun — COMMENTED** [review]
Couple questions

File: apps/web/modules/components/entity/readable-entity-page.tsx (hunk 3 above)
Comment (makeitrein) [reply apps/web/modules/components/entity/readable-entity-page.tsx:98]: Ensuring that triples are sorted the same way in both edit mode and read mode

**Review by baiirun — APPROVED** [review]

**Pull request merged**